packages/app/src/debug/DebugNewRelevantRunBar.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div
+    v-if="data.status && data.runNumber"
+    data-cy="newer-relevant-run"
+    class="w-full px-24px pb-16px gap-y-1"
+  >
+    <ul
+      id="metadata"
+      class="rounded flex flex-row bg-indigo-50 border-indigo-100 p-12px gap-x-2 items-center whitespace-nowrap children:flex children:items-center"
+    >
+      <li>
+        <DebugRunNumber
+          :status="data.status"
+          :value="data.runNumber"
+        />
+      </li>
+      <li class="font-medium text-sm text-gray-800 truncate">
+        {{ data.commitInfo?.summary }}
+      </li>
+      <li class="font-normal text-sm truncate">
+        <DebugPendingRunCounts
+          v-if="data.status === 'RUNNING'"
+          :total-skipped="data.totalSkipped || 0"
+          :total-failed="data.totalFailed || 0"
+          :total-passed="data.totalPassed || 0"
+          :total-tests="data.totalTests || 0"
+        />
+        <button
+          v-else
+          class="cursor-pointer text-indigo-500 hocus-link hocus-link-default"
+          @click="navigateToNewerRun"
+        >
+          {{ t('debugPage.viewRun') }}
+        </button>
+      </li>
+    </ul>
+  </div>
+</template>
+<script lang="ts" setup>
+import { computed } from 'vue'
+import type { DebugNewRelevantRunBarFragment } from '../generated/graphql'
+import { gql } from '@urql/core'
+import { useI18n } from 'vue-i18n'
+import DebugPendingRunCounts from './DebugPendingRunCounts.vue'
+import DebugRunNumber from './DebugRunNumber.vue'
+
+gql`
+fragment DebugNewRelevantRunBar on CloudRun {
+  id
+  runNumber
+  status
+  url
+  totalSkipped
+  totalFailed
+  totalPassed
+  totalTests
```

Comment:
These fields are counts of tests and this component should be reflecting counts of specs.  Right now the Cloud is not providing this data summarized so you would have to get the `specs` field from the `CloudRun` and group by the `status` field.

Reply:
Addressed in [`822015a`](https://github.com/cypress-io/cypress/pull/25282/commits/822015a319f0403c2c1ee5303226c9764227ad83)